### PR TITLE
Wrote tests for getCode, getGasPrice,  getMaxPriorityFee

### PR DESCRIPTION
**Pull request by proxima424**

### CODE REVIEW

File: test/e2e/ethereum-client/EthereumClient.test.ts [hunk 1]
```diff
@@ -1,7 +1,13 @@
 import { EthereumClient } from "../../../src";
-import networkConfig from "./networkConfig";
+import {networkConfig,uniNetworkConfig} from "./networkConfig";
 
 const testAddress = '0x229F701345b239c9C0a0685ef32e34842eDd1a45';
+// truncated non-20 bytes string
+const invalidTestAddress = '0x229F701345b239c9C0a0685ef32e34842';
+//
```
Comment (tomarsachin2271) [inline test/e2e/ethereum-client/EthereumClient.test.ts:7]: Either add these inline comments or remove //

File: test/e2e/ethereum-client/EthereumClient.test.ts [hunk 2]
```diff
@@ -15,7 +21,63 @@ describe('EthereumClient E2E tests', () => {
...
+
     });
+    uniNetworkConfig.forEach(({ nodeUrls })=>{
+        describe(`Ethereum E2E state tests for ${nodeUrls}`,()=>{
+            const client = new EthereumClient({defaultNodeUrls: nodeUrls});
+
+            it('should call the getCode method and return the result', async()=>{
+                const address = univ3factoryAddress;
+                const code = await client.getCode(address);
+                expect(typeof code).toBe('string');
```
Comment (tomarsachin2271) [inline test/e2e/ethereum-client/EthereumClient.test.ts:65]: Add another assertion to test the code should not be an empty string.

File: test/e2e/ethereum-client/networkConfig.ts [hunk 3]
```diff
@@ -13,4 +13,11 @@ const networkConfig: NetworkConfig[] = [
     { nodeUrls: ['https://rpc.ankr.com/polygon_mumbai'] }
 ];
 
-export default networkConfig;
+export  const uniNetworkConfig: NetworkConfig[] = [
```
Comment (tomarsachin2271) [inline test/e2e/ethereum-client/networkConfig.ts:16]: Rename it to uniswapNetworkConfig

**Review by tomarsachin2271 — APPROVED** [review]
